sops.el

```diff
@@ -65,8 +65,8 @@
   :group 'sops
   :lighter " sops"
 
-  (if (eq sops--sops-version-greater-than-equal-to-3-9 -1)
-      (sops--version-check))
+  (when (= sops--sops-version-greater-than-equal-to-3-9 -1)
```

Comment:
`(when (eq ... -1)` fails when `sops--sops-version-greater-than-equal-to-3-9` is `t` or `nil`.

We should change it back to `eq` for now.

Reply:
I see. You are right. `=` only supports numeric comparison.